src/main/java/net/javadiscord/javabot/util/StringUtils.java

```diff
@@ -109,4 +119,66 @@ public static String capitalize(String word) {
 		if (word == null || word.isEmpty()) return word;
 		return word.substring(0, 1).toUpperCase() + word.substring(1);
 	}
+
+	/**
+	 * Tries to get the current operating system.
+	 *
+	 * @return The current operating system, as a {@link String}.
+	 */
+	public static String getOperatingSystem() {
+		String os = System.getProperty("os.name");
+		if(os.equals("Linux")) {
+			try {
+				String[] cmd = {"/bin/sh", "-c", "cat /etc/*-release" };
+				Process p = Runtime.getRuntime().exec(cmd);
+				BufferedReader bri = new BufferedReader(new InputStreamReader(p.getInputStream()));
```

Comment:
Use a `try-with-resources` block in order to close this resource.

Reply:
Do they even need to be closed? The InputStreamReader is never bind to a variable

Reply:
Closing the `BufferedReader` would cascade to closing the `InputStreamReader` and the `InputStream` from the `Process`.  Therefore, closing `bri` with a `try-with-resources` block would be sufficient.

While closing them isn't strictly necessary in this case, I would still consider it a best-practice to close all resources unless it causes issues.